src/extension/features/general/resize-sidebar/index.js

```diff
@@ -0,0 +1,80 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { getToolkitStorageKey, setToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+/* global $, console, require */
+
+const resizerClass = 'sidebar-resizer';
```

Comment:
nit: I want to say the standard throughout the repo is that constants like are defined `UPPER_SNAKE_CASE`

Reply:
Is this one not enforced by your eslint? I am so accustomed to `camcelCase`, and Visual Studio told me `UPPER_CASE` was wrong, so I just went for it. Will correct that.